### DISCUSSION TITLE
What teams and ideas are participating in the Hack? What is the process?

### DISCUSSION THREAD

**Discussion by tellingmachine** [Green-Software-Foundation/hack#116] [Ask Anything]
I submitted an idea, but I am not certain if I am part of the competition. Does an idea need to be accepted by the blackthon organizers? How can I tell what ideas and teams are accepted to the competition?

**Comment by russelltrow** [comment]
Hey @tellingmachine,

Your project was created correctly, you simply need to take the next step of [**registering on the carbon hack website**](https://hack.greensoftware.foundation/register). This will officially register your project and give you direct access to the Impact Framework team.

Anyone is free to raise a project, there are [minimal acceptance criteria](https://github.com/Green-Software-Foundation/hack/wiki/Rules-and-Terms). The entire process is should be covered in the [Participant Guide](https://github.com/Green-Software-Foundation/hack/wiki/Participant-Guide).

The projects are all listed as [Issues](https://github.com/Green-Software-Foundation/hack/issues).
